client/keys/utils.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/tendermint/tmlibs/cli"
 	dbm "github.com/tendermint/tmlibs/db"
 
+	bech32cosmos "github.com/cosmos/bech32cosmos/go"
```

Comment:
maybe use smaller abbr. such as `bech32`?

Reply:
I was to clearly distinguish from the bitcoin variant.

Reply:
fair - I'm not super familiar with this world. Are there other implementations with bech32 besides bitcoin? do they all use the word `bech32` - how about `cesh32` hahaha. good for whatever though really, just figured it was a long import name

Reply:
* cesh32 * <- I'm a huge supporter of this name